In [57]:
import pandas as pd

In [ ]:
# df1 = pd.read_csv('Apple_test_cleaned.csv', encoding='latin-1')

In [ ]:
# df1.info()

<class 'pandas.core.frame.DataFrame'>
Index: 3725 entries, 0 to 3885
Data columns (total 3 columns):
 #   Column                Non-Null Count  Dtype  
---  ------                --------------  -----  
 0   sentiment             3725 non-null   object 
 1   sentiment:confidence  3725 non-null   float64
 2   text                  3725 non-null   object 
dtypes: float64(1), object(2)
memory usage: 116.4+ KB


In [60]:
# df1 = df1.drop(['_golden','_unit_state','_trusted_judgments','_last_judgment_at','date','id','query','sentiment_gold'], axis=1)

In [61]:
# df1 = df1.to_csv("Apple_test_cleaned.csv", index=False)

In [ ]:
# df1.isnull().sum()

sentiment               0
sentiment:confidence    0
text                    0
dtype: int64

##### The 82 rows with not_relevant sentiment have been removed. The DataFrame now only contains the relevant sentiment values:



In [ ]:
# df1 = df1[df1['sentiment'] != 'not_relevant']
# df1['sentiment'].value_counts()

sentiment
3    2162
1    1219
5     423
Name: count, dtype: int64

##### 79 rows with fewer than 5 words in the text column were removed (3804 → 3725 rows remaining). 

In [ ]:
# print("Before:", len(df1))
# df1 = df1[df1['text'].str.split().str.len() >= 5]
# print("After:", len(df1))

Before: 3804
After: 3725


In [ ]:
# df1.to_csv("Apple_test_cleaned2.csv", index=False)

In [69]:
df2 = pd.read_csv("D:\\ML CODES\\IBM\\test\\hotel test.csv")

In [72]:
df2.info()

<class 'pandas.core.frame.DataFrame'>
RangeIndex: 35912 entries, 0 to 35911
Data columns (total 19 columns):
 #   Column                Non-Null Count  Dtype  
---  ------                --------------  -----  
 0   address               35912 non-null  object 
 1   categories            35912 non-null  object 
 2   city                  35912 non-null  object 
 3   country               35912 non-null  object 
 4   latitude              35826 non-null  float64
 5   longitude             35826 non-null  float64
 6   name                  35912 non-null  object 
 7   postalCode            35857 non-null  object 
 8   province              35912 non-null  object 
 9   reviews.date          35653 non-null  object 
 10  reviews.dateAdded     35912 non-null  object 
 11  reviews.doRecommend   0 non-null      float64
 12  reviews.id            0 non-null      float64
 13  reviews.rating        35050 non-null  float64
 14  reviews.text          35888 non-null  object 
 15  reviews.title      

In [ ]:
df2 = df2.drop([])